x-pack/test/apm_api_integration/tests/service_overview/__snapshots__/instances_detailed_statistics.spec.snap

```diff
@@ -1,1317 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`APM API tests service_overview/instances_detailed_statistics.spec.ts basic apm_8.0.0 Service overview instances detailed statistics when data is loaded fetching data with comparison returns the right data for current and previous periods 5`] = `
```

Comment:
Isn't this snapshot still being used?

Reply:
Oops, deleted a few too many things. Reverting this one xD